service/api_logic/interactions_logic.py

```diff
@@ -0,0 +1,56 @@
+from dto.api_output import OutputInteractions
+from service.api_logic.models.api_models import InteractionTypes
+from database.postgres.dto import InteractionWithNewsDTO
+from exept.exeptions import ArticleNotFoundError
+from logger.logger import Logger
+from datetime import datetime, timezone
+
+LIKE_ID = InteractionTypes['LIKE'].value
+DISLIKE_ID = InteractionTypes['DISLIKE'].value
+
+class InteractionWithNewsService:
+    def __init__(self, interaction_with_news_dal, news_dal):
+        self._interaction_with_news_dal = interaction_with_news_dal
+        self._news_dal = news_dal
+
+        self._logger = Logger("logger", "all.log").logger
+
+    def __convert_input_to_db_dto(self, interaction_input_dto):
```

Comment:
MINOR:

You should always try to specify the parameter's type. That'll semplify both code production and readability.

(e.g.  param: string, param2: int)